types/globalsat/LS-113G/default.schema.json

```diff
@@ -0,0 +1,48 @@
+{
+  "$id": "https://globalsat/LS-113G-default.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Default",
+  "type": "object",
+  "properties": {
+    "humidity": {
+      "title": "Humidity",
+      "unit": "%",
+      "type": "number",
+      "description": "The relative humidity in %",
+      "minimum": 0,
+      "maximum": 100
+    },
+    "temperature": {
+      "title": "Temperature",
+      "unit": "°C",
+      "type": "number",
+      "description": "The temperature in °C.",
+      "minimum": -3276.5,
+      "maximum": 3276.5
+    },
+    "co2": {
+      "title": "CO2",
+      "unit": "ppm",
+      "type": "integer",
+      "description": "Carbon Dioxide in ppm",
+      "minimum": 0,
+      "maximum": 10000
+    },
+    "co": {
```

Comment:
change it to co2?

Reply:
![image](https://user-images.githubusercontent.com/73245684/110969711-4e110b80-8359-11eb-8b78-a39af7339b0b.png)
It's CO ^^

Reply:
then the description is wrong